finalPrep/annotations.js

```diff
@@ -2,41 +2,69 @@
     clothes: clothes,
+    // The exterior key has the value of the buildABear clothes parameter.
     exterior: fur,
+    // The cost key as a fixed integer value.
     cost: 49.99,
+    // The sayings key is assigned the value of an array with three elements: the above greeting and powerSaying variables as well as a new, fixed string for saying goodnight.
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+    // The isCuddly key has a fixed boolean value of true.
     isCuddly: true,
   };
 
+  // The return keyword enables the computer to output the above builtObject variable, aka the 6 property object, as the result of executing this function.
   return builtBear
 }
 
+// Calls the buildABear function for the first time. Elements within parantheses are the function arguments- a string, integer, string, array, and string, respectively. These arguments take the place of the 5 parameters we identified upon declaring the function. They occupy the various places where the parameters were used within the function. For example, the greeting variable will fill the ${name} section with "Fluffy" to have an end value of `Hey partner! My name is Fluffy - will you be my friend?!` The end result of calling the function will be a builtBear object with all parameters updated with the below values.
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+// Calls the buildABear function for a second time with all new arguments that. The builtBear object is returned with all new values.
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
+
+// Declares a new function called 'fizzBuzz' with 3 parameters- num1, num2, and range
 function fizzBuzz(num1, num2, range) {
+  // Starts a for loop. The conditions of this for loop are that (1) the initial counter variable `i` will start at 0. It is 0 the first time the loop runs. (2) the condition in order to keep looping is that the counter variable `i` must be less than or equal to the range parameter that we pass in from the function declaration parantheses. Once `i` equals the range parameter, it will exit the loop. Finally, (3) the third part of the condition is that at the end of each loop iteration, the counter variable `i` increases by one.
   for (var i = 0; i <= range; i++) {
+    // Sets up a conditional statement that checks two conditions. The && sign means that both conditions must be true in order for this first condition to be met. Condition #1 evaluates if the modulo, or remainder, of our counter variable `i` divided by the num1 parameter is strictly equal to 0. Condition #2 evaluates whether the same is true but for the modulo of `i` divided by the num2 parameter. Only if both modulos are 0 does the code in the curly bracket execute at which time the computer also exits the conditional statement and does not evaluate any below conditions. Otherwise, if not met, it moves on to the next condition.
```

Comment:
Nice explanations. You have clearly digested what a modulo is, as well as comparative operators. Love to see the practice with technical explaination.